docs/storage-backend/cassandra.md

```diff
@@ -194,37 +194,62 @@ iterate over all vertices or edges in large graphs effectively.
 >
 > —  [DataStax Astra](https://www.datastax.com/products/datastax-astra)
 
-[Download](https://docs.datastax.com/en/astra/aws/doc/dscloud/astra/dscloudObtainingCredentials.html) the secure-connect zipped bundle for your Astra database.
-
-Unzip your copy of `secure-connect-your_astra_db.zip` which will contain the following files:
-
-- `ca.crt`
-- `cert`
-- `cert.pfx`
-- `config.json`
-- `cqlshrc`
-- `identity.jks`
-- `key`
-- `trustStore.jks`
-
-The `config.json` file contains the following:
-
-- `host` - the contact point for the CQL driver
-- `port` - the CQL client port
-- `keyspace` - the name of the keyspace you created
-- `trustStoreLocation` - trustStore.jks is the truststore
-- `trustStorePassword` - your truststore password
-- `keyStoreLocation` - identity.jks is the keystore
-- `keyStorePassword` - your keystore password
-
-Using the information above, you will need to configure the following:
-
-- `storage.hostname`
-- `storage.port`
-- `storage.username`
-- `storage.password`
-- `storage.cql.keyspace`
-- `storage.cql.ssl.*`
+[Download](https://docs.datastax.com/en/astra/docs/obtaining-database-credentials.html) the secure-connect zipped bundle for your Astra database.
+
+While connecting to Astra DB from JanusGraph, it is preferred to make use of the secure bundle connection file
+as-is without extracting it. There are multiple ways in which a secure bundle connection file can be passed on to
+the JanusGraph configuration to connect to Astra DB using the DataStax driver.
+
+### Internal string configuration
+Set the following property `storage.cql.internal.string-configuration` to `datastax-java-driver { basic.cloud.secure-connect-bundle=<path-to-secure-bundle-zip-file> }`
+and set the username, password and keyspace details
+
+For eg.
```

Comment:
Please replace "For eg." with "For example:"

"e.g." already means "for example" – see https://en.wiktionary.org/wiki/e.g. – so "for eg." means "for for example".